### DISCUSSION TITLE
[Question] Confused on use-case/scenario for UWP apps

### DISCUSSION THREAD

**Discussion by dmitrymal** [microsoft/WindowsAppSDK#534] [Q&A]
We are developing UWP Apps that we distribute through the Windows 10 App Store. ProjectReunion was brought to our attention as something that could provide additional help/simplify our development efforts.

Reading through FAQs and Wikis on this site, we are still confused on what would this project bring to the table and how would it simplify our coding experience.

**Could someone give a couple simple examples where this project would improve/simplify UWP development?**

**Another question/concern, does it mean that the UWP itself is being phased out in favor of these unified Apis?** For example, what would really help us is to have support for .net standard 2.1  in UWP apps. However, this has been in TBD for years: 
https://docs.microsoft.com/en-us/dotnet/standard/net-standard **Any insight on this? Is it part of ProjectReunion?**

**Comment by MattBDev** [comment]
From what I understand UWP is not being phased out.

**Comment by shaheedmalik** [comment]
Win32 = Desktop 
UWP = Entire Microsoft ecosystem.

**Comment by mandalorianbob** [comment]
From https://github.com/microsoft/ProjectReunion/discussions/462 it appears that UWP apps will not be supported in .NET 6. The only way to target UWP in .NET 6 is to use one of the non-UWP frameworks - i.e. Xamarin, MAUI, Blazor, etc, targeting UWP. UWP itself, i.e. a C# Win 10 UWP App, will not be supported.

**Reply by shaheedmalik** [reply]
> From #462 it appears that UWP apps will not be supported in .NET 6. The only way to target UWP in .NET 6 is to use one of the non-UWP frameworks - i.e. Xamarin, MAUI, Blazor, etc, targeting UWP. UWP itself, i.e. a C# Win 10 UWP App, will not be supported.

More like UWP hasn't been updated to use WinUI 3 yet, if I remember correctly.
